spyder_line_profiler/spyder/widgets.py

```diff
@@ -100,123 +112,217 @@ def natural_sort_key(key):
-        #self.filecombo.valid.connect(self.show_data)
-        # FIXME: The combobox emits this signal on almost any event
-        #        triggering show_data() too early, too often.
-
-        browse_button = create_toolbutton(
-            self, icon=get_icon('fileopen.png'),
-            tip=_('Select Python script'),
-            triggered=self.select_file)
-
-        self.datelabel = QLabel()
-
-        self.log_button = create_toolbutton(
-            self, icon=get_icon('log.png'),
-            text=_("Output"),
-            text_beside_icon=True,
+            icon=self.create_icon('stop'),
+            triggered=self.kill_if_running,
+        )
+        self.browse_action = self.create_action(
+            SpyderLineProfilerWidgetActions.Browse,
+            text=_("Open Script"),
+            tip=_('Select Python script'),            
+            icon=self.create_icon('fileopen'),
+            triggered=self.select_file,
+        )
+        self.log_action = self.create_action(
+            SpyderLineProfilerWidgetActions.ShowOutput,
+            text=_("Show Result"),
             tip=_("Show program's output"),
-            triggered=self.show_log)
-
-        self.datatree = LineProfilerDataTree(self)
-
-        self.collapse_button = create_toolbutton(
-            self,
-            icon=get_icon('collapse.png'),
+            icon=qta.icon("fa.file-text", color=ima.MAIN_FG_COLOR),
+            triggered=self.show_log,
+        )
+        self.collapse_action = self.create_action(
+            SpyderLineProfilerWidgetActions.Collapse,
+            text=_("Collaps"),
+            tip=_('Collapse all'),
+            icon=qta.icon("mdi.arrow-collapse-all", color=ima.MAIN_FG_COLOR),
             triggered=lambda dD=-1: self.datatree.collapseAll(),
-            tip=_('Collapse all'))
-        self.expand_button = create_toolbutton(
-            self,
-            icon=get_icon('expand.png'),
-            triggered=lambda dD=1: self.datatree.expandAll(),
-            tip=_('Expand all'))
-
-        hlayout1 = QHBoxLayout()
-        hlayout1.addWidget(self.filecombo)
-        hlayout1.addWidget(browse_button)
-        hlayout1.addWidget(self.start_button)
-        hlayout1.addWidget(self.stop_button)
-
-        hlayout2 = QHBoxLayout()
-        hlayout2.addWidget(self.collapse_button)
-        hlayout2.addWidget(self.expand_button)
-        hlayout2.addStretch()
-        hlayout2.addWidget(self.datelabel)
-        hlayout2.addStretch()
-        hlayout2.addWidget(self.log_button)
-
-        layout = QVBoxLayout()
-        layout.addLayout(hlayout1)
-        layout.addLayout(hlayout2)
-        layout.addWidget(self.datatree)
-        self.setLayout(layout)
+        )
+        self.expand_action = self.create_action(
+            SpyderLineProfilerWidgetActions.Expand,
+            text=_("Expand"),
+            tip=_('Expand all'),
+            icon=qta.icon("mdi.arrow-expand-all", color=ima.MAIN_FG_COLOR),
```

Comment:
This should be:
```suggestion
            icon=self.create_icon('expand'),
```
Which will make it look like this:
![imagen](https://user-images.githubusercontent.com/16781833/159551439-8213fbcf-8cee-4963-8293-59f1e53ef0c1.png)